packages/dcos-net/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "dcos-net": {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-net.git",
-      "ref": "b02814510c1d4686b7055c4f70f44410bc3892c9",
+      "ref": "03f249b1c5f1d8a12b571a35ffa34efa07c52375",
```

Comment:
Have you considered tagging the releases so it makes easier to track down what version of dcos-net is shipped with every dcos bump?

Reply:
@Fabs I am not sure if pkgpanda supports it. Besides that we have here the `master` branch of DC/OS reffering to the tip of the `master` branch of DC/OS net. The way it is handled e.g. for bouncer is that e.g. `1.12` branch of DC/OS reffers to the tip fo the `1.12` branch of Bouncer. I do not know if tags would be of any help in our release model.